src/api/resourceApi.ts

```diff
@@ -11,11 +11,52 @@ import {
 import { AccessTypes, License } from '../types/license.types';
 import { Content, emptyResourceContent, LinkMetadataFilename } from '../types/content.types';
 import { authenticatedApiRequest } from './api';
-import { QueryObject, SearchResult } from '../types/search.types';
+import { QueryObject, SearchParameters, SearchResult } from '../types/search.types';
 
-export const searchResources = (query: QueryObject): Promise<AxiosResponse<SearchResult>> => {
-  let url = `${API_PATHS.guiBackendResourcesSearchPath}/resources/search?query=${query.query}&limit=${query.limit}`;
-  if (query.offset > 0) url += `&offset=${query.offset}`;
+export const searchResources = ({
+  query,
+  limit,
+  institutions,
+  resourceType,
+  licenses,
+  keywords,
+  offset,
+}: QueryObject): Promise<AxiosResponse<SearchResult>> => {
+  let url = `${API_PATHS.guiBackendResourcesSearchPath}/resources/search?query=${query}`;
+  if (
+    (institutions && institutions.length > 0) ||
+    resourceType.length > 0 ||
+    licenses.length > 0 ||
+    keywords.length > 0
+  ) {
+    url += '&filter=';
+    const filters: string[] = [];
+    if (institutions.length > 1) {
+      filters.push(`facet_institution::(${institutions.join(' OR ')})`);
+    } else if (institutions.length === 1) {
+      filters.push(`facet_institution::${institutions[0]}`);
+    }
```

Comment:
blir ikke dette det samme? hvis du tar join med bare ett elment får du bare selve elementet
```suggestion
    if (institutions.length > 0) {
      filters.push(`facet_institution::(${institutions.join(' OR ')})`);
    }
```

Reply:
ups. tilbakestilt

Reply:
Oi, ser også at vi trenger å konverte institusjonene til lowercase for at det skal fungere

Reply:
Så enten ta tabellen med institusjoner å konverter til lowercase i metoden, eller sørg for at den blir sent til api'et som lowercase